support-frontend/assets/pages/digital-subscription-checkout/components/digitalCheckoutFormGift.jsx

```diff
@@ -0,0 +1,298 @@
+      'Sorry we could not process your payment';
+    return (
+      <Content>
+        <CheckoutLayout aside={(
+          <OrderSummary
+            image={
+              <GridImage
+                gridId="subscriptionDailyPackshot"
+                srcSizes={[1000, 500]}
+                sizes="(max-width: 740px) 50vw, 500"
+                imgType="png"
+                altText=""
+              />
+                }
+            title="Digital Gift Subscription"
+            description="Premium App + The Guardian Daily + Ad-free"
+            productPrice={productPrice}
+            billingPeriod={props.billingPeriod}
+            changeSubscription={routes.digitalSubscriptionLanding}
+            orderIsAGift
+          />)}
+        >
+          <Form onSubmit={(ev) => {
+                ev.preventDefault();
+                props.submitForm();
+              }}
+          >
+            <FormSection title="Gift recipient's details">
+              <PersonalDetailsDigitalGift
+                firstNameGiftRecipient={props.firstNameGiftRecipient || ''}
+                setFirstNameGift={props.setFirstNameGift}
+                lastNameGiftRecipient={props.lastNameGiftRecipient || ''}
+                setLastNameGift={props.setLastNameGift}
+                emailGiftRecipient={props.emailGiftRecipient || ''}
+                setEmailGift={props.setEmailGift}
+                formErrors={((props.formErrors: any): FormError<PersonalDetailsFormField>[])}
+              />
+            </FormSection>
+            <FormSection title="Gift delivery date">
+              <DatePickerWithError
+                id="giftDeliveryDate"
+                error={firstError('giftDeliveryDate', props.formErrors)}
+                value={props.giftDeliveryDate}
+                onChange={date => props.setDigitalGiftDeliveryDate(date)}
+              />
+            </FormSection>
+            <FormSection title="Personalise your gift">
+              <TextAreaWithLabel
+                id="gift-message"
+                label="Gift message"
+                maxlength={300}
+                value={props.giftMessage}
+                setValue={props.setGiftMessage}
+                optional
+              />
+            </FormSection>
+            <FormSection title="Your details" border="top">
+              <PersonalDetails
+                firstName={props.firstName}
+                setFirstName={props.setFirstName}
+                lastName={props.lastName}
+                setLastName={props.setLastName}
+                email={props.email}
+                telephone={props.telephone}
+                setTelephone={props.setTelephone}
+                formErrors={props.formErrors}
+                signOut={props.signOut}
+              />
+            </FormSection>
+            <FormSection title="Billing address">
+              <Address />
```

Comment:
I think this is an existing bug so probably needs a new card, but if you enter a postcode that isn't found, this error message is not announced to a screen-reader user- very confusing!

![Screenshot 2020-09-22 at 15 02 07](https://user-images.githubusercontent.com/29146931/93893737-e4873d00-fce5-11ea-8f34-fd216871d8fd.png)

Reply:
Yes, that isn't great. Shall see if I can fix it.